packages/bot-web-ui/src/components/dashboard/quick-strategy/quick-strategy.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { connect } from 'Stores/connect';
+import RootStore from 'Stores/index';
+import { TQuickStrategyProps } from './quick-strategy.types';
+import { QStrategyContainer } from './quick-strategy-components';
+import { localize } from '@deriv/translations';
+import { MobileFullPageModal } from '@deriv/components';
+
+const QuickStrategy = (props: TQuickStrategyProps) => {
+    const { is_mobile, is_strategy_modal_open, toggleStrategyModal } = props;
+    React.useEffect(() => {
+        toggleStrategyModal();
+        // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
do we need this?

Reply:
better not to avoid getting errors with disabling the eslint rule, because they are warning the mistakes. Is it possible to read this and apply changes accordingly to solve this issue? 
https://bobbyhadz.com/blog/react-hooks-exhaustive-deps#:~:text=The%20%22react%2Dhooks%2Fexhaustive,render%20or%20disable%20the%20rule.

Reply:
we really need empty dependency, because we want to render only when the component has been loaded/mounted